tests/system/small/ml/test_impute.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pandas as pd
+
+from bigframes.ml import impute
+import bigframes.pandas as bpd
+
+
+def test_simple_imputer_normalized_fit_transform_default_params():
+    missing_df = bpd.DataFrame(
```

Comment:
The input is used in multiple tests, we can extract to a test fixture in this file.

Reply:
Done.